components/ArticleList.jsx

```diff
@@ -22,10 +23,12 @@ const ArticleList = () => {
       <ul className="article-card-list">
         {articleList.map(article => {
           return (
-            <ArticleCard
-            key={article.article_id}
-            {...article}
-            />  
+            <Link to={`/articles/${article.article_id}`}>
```

Comment:
This is technically not semantic HTML. The only children in a UL or OL should be an LI. Currently your structure is like so:
```
<ul>
  <a>
    <li></li>
  </a>
</ul>
```